### DISCUSSION TITLE
Two accounts, one domain

### DISCUSSION THREAD

**Discussion by hermanboxcar5** [vercel/community#5414] [Help] [answered]
### Question

I am part of a nonprofit organization, I am using vercel to host one of our assets that we run for our team. However, there is also another person using their account on vercel (same job as me) who hosts our blog on vercel. The problem is, I need to register the DNS for vercel to link our website (say dev.example.com) to a vercel project, but he is also using blog.example.com for the blog. When I tried to link dev.example.com, Vercel gave me an error:
```
Another Vercel account is using this domain. Please set the following TXT record on _vercel.example.org to use dev.example.org in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.
```

**Comment by amyegan** [comment, marked as answer]
Hey @hermanboxcar5. If you add the TXT record to [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access), then you can use the subdomain with your project even if the main domain is owned by the other team.
